lib/vfvalue.h

```diff
@@ -280,27 +299,12 @@ namespace ValueFlow
 
         ErrorPath errorPath;
 
-        ErrorPath debugPath;
+        ErrorPath debugPath; // TODO: make lighter by default
 
         /** For calculated values - varId that calculated value depends on */
         nonneg int varId{};
 
-        /** value relies on safe checking */
-        bool safe{};
-
-        /** Conditional value */
-        bool conditional{};
-
-        /** Value is is from an expanded macro */
-        bool macro{};
-
-        /** Is this value passed as default parameter to the function? */
-        bool defaultArg{};
-
-        int indirect{};
-
-        /** kind of moved  */
-        enum class MoveKind : std::uint8_t { NonMovedVariable, MovedVariable, ForwardedVariable } moveKind = MoveKind::NonMovedVariable;
+        int indirect{}; // TODO: can we reduce the size?
```

Comment:
@chrchr-github how high can this go? would a 8-bit value be enough for this?

Reply:
8bit unsigned should plenty, unless negative values are used somewhere?

Reply:
It looks like it can go negative based on the code so it should be signed.